README.md

```diff
@@ -4,7 +4,7 @@
 
 ## Overview
 
-appleseed-maya is an appleseed plugin for [Autodesk® Maya®](http://www.autodesk.com/products/maya/overview) 2017 and 2018.
+appleseed-maya is an appleseed plugin for [Autodesk® Maya®](http://www.autodesk.com/products/maya/overview) 2018, 2019 and 2020.
```

Comment:
We probably should clarify that 2018 is not supported on Linux:

    ... for Maya 2018 (Windows only), 2019 and 2020.

Reply:
Done